starter-core/src/main/java/io/micronaut/starter/feature/other/HttpClient.java

```diff
@@ -30,12 +37,37 @@
 @Singleton
 public class HttpClient implements DefaultFeature {
 
+    private final RxJavaTwoHttpClient rxJavaTwoHttpClient;
+    private final RxJavaThreeHttpClient rxJavaThreeHttpClient;
```

Comment:
Can we rename the classes to `RxJava2HttpClient`, RxJava3HttpClient` (and also the other one `RxJava1HttpClient`

Reply:
It does not matter right?

Reply:
No, we can name the classes whatever we want but I think we should be consistent with the names. We've always named them with numbers not with letters.

Reply:
The class `RxJavaThree` already existed. That it is the reason I closed `RxJavaThreeHttpClient`. I can rename every class if you feel strong about this.